spec/unit/provider/group/dscl_spec.rb

```diff
@@ -131,6 +136,7 @@
 
     it "should return false for an unused gid number" do
       expect(@provider.gid_used?(501)).to be_falsey
+      expect(@provider.gid_used?(18)).to be_falsey
```

Comment:
as I mentioned before here, interesting tests with this particular output are if 50, 5000, 00, 5, or 0 return falsey, since that's the only real logic your function does - check that the output contains the gid you searched for, checking that's actually valid (so say someone else doesn't break it later) is a really good test. I can see someone checking that it say it *contains* the number or *startswith* the number (by accident).

That's why I suggested a moving the output to the before block so you can run all these various tests against that output.